service/src/NVMLIOGroup.hpp

```diff
@@ -107,6 +107,7 @@ namespace geopm
                 std::vector<std::shared_ptr<signal_s> > signals;
                 int domain;
                 std::function<double(const std::vector<double> &)> m_agg_function;
+                int m_behavior;
```

Comment:
In general, we do not add member functions to structs (unlike classes).  For this reason, we do not typically prefix struct fields with `m_`.  Please be consistent in this naming.